commit 95baba5b98efc2ed2716e85e5e6d3d779efe0a95 (refs/changes/54/754/1)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2016-10-20T15:35:42-07:00 (3 years ago)
    
    ADMINUI-2349 VM Detail Page breaks when disk does not have image_uuid #204

diff --git a/lib/api/page-vm.js b/lib/api/page-vm.js
index 5111558a..9ee2decd 100644
--- a/lib/api/page-vm.js
+++ b/lib/api/page-vm.js
@@ -22,7 +22,7 @@ function getPageVmData(req, res, next) {
         if (vm.image_uuid) {
             imageUuid = vm.image_uuid;
         } else {
-            imageUuid = vm.disks && vm.disks.length ? vm.disks[0].image_uuid : '';
+            imageUuid = vm.disks && vm.disks.length && vm.disks[0].image_uuid ? vm.disks[0].image_uuid : '';
         }
 
         async.parallel([
